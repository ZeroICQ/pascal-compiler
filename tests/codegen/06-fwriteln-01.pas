begin
    fwriteln(20);
    fwriteln(1.2);
    fwriteln(123.3e-12);
end.