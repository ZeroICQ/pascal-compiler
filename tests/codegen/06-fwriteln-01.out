20
1.2
1.233e-010
